x-pack/legacy/plugins/beats_management/types/common.ts

```diff
@@ -4,9 +4,4 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-declare module '*.png';
-
-declare module '*.svg' {
-  const content: string;
-  export = content;
-}
+import {} from '../../reporting/public/components/report_listing';
```

Comment:
Don't think this is needed anymore, I moved the declaration that it needed to `x-pack/typings`.